Can't build livy-assembly: gives user id '2074918846' is too big

Could not build livy from master:

[INFO] Reactor Summary:
[INFO]
[INFO] livy-main .......................................... SUCCESS [ 30.984 s]
[INFO] livy-api_2.10 ...................................... SUCCESS [03:32 min]
[INFO] livy-client-common_2.10 ............................ SUCCESS [  1.972 s]
[INFO] livy-client-local_2.10 ............................. SUCCESS [ 19.896 s]
[INFO] livy-core_2.10 ..................................... SUCCESS [ 36.403 s]
[INFO] livy-repl_2.10 ..................................... SUCCESS [01:11 min]
[INFO] livy-yarn_2.10 ..................................... SUCCESS [ 11.750 s]
[INFO] livy-spark_2.10 .................................... SUCCESS [ 26.428 s]
[INFO] livy-server_2.10 ................................... SUCCESS [ 34.198 s]
[INFO] livy-assembly_2.10 ................................. FAILURE [  9.494 s]
[INFO] livy-client-http_2.10 .............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 07:35 min
[INFO] Finished at: 2016-01-08T19:28:50-07:00
[INFO] Final Memory: 47M/11135M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.6:single (livy-assembly) on project livy-assembly_2.10: Execution livy-assembly of goal org.apache.maven.plugins:maven-assembly-plugin:2.6:single failed: user id '2074918846' is too big ( > 2097151 ). -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.


$ id -u
2074918846

We have our corporate UNIX user ids synchronized with an Active Directory, so they are unique across all servers and .. long. Any idea of a workaround to get Livy Server compiled?

--- a/assembly/pom.xml
+++ b/assembly/pom.xml
@@ -87,6 +87,9 @@
             </goals>
             <configuration>
               <finalName>${assembly.finalName}</finalName>
+              <formats>
+                <format>zip</format>
+              </formats>
             </configuration>
           </execution>
         </executions>

